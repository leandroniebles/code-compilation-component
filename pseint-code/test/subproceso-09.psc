subproceso lala
finsubproceso
subproceso lala
finsubproceso
