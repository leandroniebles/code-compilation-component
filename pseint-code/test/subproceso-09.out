=== Line 3: SynError 243
=== Line 1: SynError 273
